### ISSUE TITLE
Hitting enter to see commit detail will highlight all space character

### ISSUE THREAD

**Issue by AchmadFathoni** [rbong/vim-flog#83]
Step :
1. Run `:Flog`
2. Hit enter.
3. Every space characters will be highlighted.
4. Need to run `:noh`or hit enter again to remove highlight.
5. Move the cursor to other commit & hit enter again will repeat the issue.

Minimal Configuration :
```
" Plugins will be downloaded under the specified directory.
call plug#begin('~/.config/nvim/plugged')
" Declare the list of plugins.
Plug 'tpope/vim-fugitive'
Plug 'rbong/vim-flog'
call plug#end()
```
Version : 
fb137707b84a328d3f1b53d22894e3c52f4a6ddb

**Comment by rbong** [comment]
I can only kind of recreate this:
1. Run `:Flog`
2. Search for anything (ex. `/\s`)
3. Run `:noh`
4. Hit enter
5. Every space character is highlighted

Running `:noh` again does fix the issue - but so does `:redraw!`, meaning nothing should actually be highlighted and it is some kind of neovim highlight bug. May even be terminal specific, not sure.

Are you able to clear the highlight with `:redraw!`? If so, it is probably related to the same bug as the one I am experiencing and it is probably a neovim bug.

**Comment by AchmadFathoni** [comment]
It fixed in Neovim 0.7.2-2 update on Archlinux
